<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1522</width>
    <height>825</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="DrawLabel" name="drawLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>1201</width>
      <height>801</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>290</width>
      <height>495</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="10" column="0">
      <widget class="QLabel" name="clipperColorLabel">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QSpinBox" name="x2SpinBox">
       <property name="maximum">
        <number>1200</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="xSpinBox">
       <property name="maximum">
        <number>1200</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="x1SpinBox">
       <property name="maximum">
        <number>1200</number>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="clippedLineColorLabel">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="5">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="xTextLabel">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="5">
      <widget class="QPushButton" name="addClipperVertexPushButton">
       <property name="text">
        <string>Add clipper vertex</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="5">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="16" column="0" colspan="5">
      <widget class="QPushButton" name="deleteClipperPushButton">
       <property name="text">
        <string>Delete clipper</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0" colspan="5">
      <widget class="QPushButton" name="closeClipperPushButton">
       <property name="text">
        <string>Close clipper</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0" colspan="5">
      <widget class="QPushButton" name="clearAllPushButton">
       <property name="text">
        <string>Clear all</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="x1TextLabel">
       <property name="text">
        <string>x1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="x2TextLabel">
       <property name="text">
        <string>x2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QSpinBox" name="y2SpinBox">
       <property name="maximum">
        <number>800</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="yTextLabel">
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="ySpinBox">
       <property name="maximum">
        <number>800</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="lineColorLabel">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="5">
      <widget class="QPushButton" name="clipPushButton">
       <property name="text">
        <string>Clip</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="y1TextLabel">
       <property name="text">
        <string>y1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="y1SpinBox">
       <property name="maximum">
        <number>800</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="y2TextLabel">
       <property name="text">
        <string>y2</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="5">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="1" colspan="4">
      <widget class="QPushButton" name="setClippedLineColorPushButton">
       <property name="text">
        <string>Set clipped line color</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="5">
      <widget class="QPushButton" name="addLinePushButton">
       <property name="text">
        <string>Add line</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1" colspan="4">
      <widget class="QPushButton" name="setClipperColorPushButton">
       <property name="text">
        <string>Set clipper color</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="4">
      <widget class="QPushButton" name="setLineColorPushButton">
       <property name="text">
        <string>Set line color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="2" rowspan="2">
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawLabel</class>
   <extends>QLabel</extends>
   <header>drawlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
